rdr_service/dao/genomics_dao.py

```diff
@@ -4340,16 +4341,31 @@ def get_data_ready_for_w1il_manifest(self, module: str, cvl_id: str, sample_ids=
         informing_loop_subquery = aliased(GenomicInformingLoop, informing_loop_decision_query.subquery())
         pipeline_metrics = aliased(GenomicGCValidationMetrics)
 
+        def get_archive_path(*, file_path):
+            return func.replace(file_path, gc_site_data_bucket, gc_site_data_bucket + '-archive')
+
         with self.session() as session:
             query = session.query(
                 func.concat(get_biobank_id_prefix(), GenomicSetMember.biobankId),
                 GenomicSetMember.sampleId,
-                GenomicGCValidationMetrics.hfVcfPath.label('vcf_raw_path'),
-                GenomicGCValidationMetrics.hfVcfTbiPath.label('vcf_raw_index_path'),
-                GenomicGCValidationMetrics.hfVcfMd5Path.label('vcf_raw_md5_path'),
-                GenomicGCValidationMetrics.gvcfPath.label('gvcf_path'),
-                GenomicGCValidationMetrics.gvcfMd5Path.label('gvcf_md5_path'),
-                GenomicGCValidationMetrics.cramPath.label('cram_name'),
+                func.IF(and_(gc_site_id == 'uw', GenomicGCValidationMetrics.pipelineId.ilike('dragen_3.4.12')),
+                        get_archive_path(file_path=GenomicGCValidationMetrics.hfVcfPath),
+                        GenomicGCValidationMetrics.hfVcfPath).label('vcf_raw_path'),
+                func.IF(and_(gc_site_id == 'uw', GenomicGCValidationMetrics.pipelineId.ilike('dragen_3.4.12')),
```

Comment:
Can we change all the refs to the gc_site_id to the param value for cvl_site_id since we know those will persist since one day we could add logic the the transform function?

Reply:
Yep, I'll update to use both config & param values!